### DISCUSSION TITLE
How to get the current progress or status of building index?

### DISCUSSION THREAD

**Discussion by flydsc** [milvus-io/milvus#3963] [Q&A and General discussion] [answered]
Hi all,

I am wondering it would be great if someone could help with indexing issues.

When we created a collection with create() function building an index at the same time, then, we insert some data.

How can we get to know the index building is done? And how we can track the progress of building index?

Best,
Shichao

**Comment by shiyu22** [comment, marked as answer]
Hi Shichao,

The get_collection_info(collection_name) function can get the information about the collection, including the index.

For more information about index, you can use sqlite or mysql to [get the meta data](https://www.milvus.io/cn/blogs/2019-12-24-view-metadata.md), and view [metadata table fields](https://www.milvus.io/cn/blogs/2019-12-27-meta-table.md).

And if you want to track the process, you can check the `engine_type` or `file_type` fields.

Sincerely.
